src/test/scala/nl.knaw.dans.easy.validatebag/rules/profile/NumberedRulesSpec.scala

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (C) 2018 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.validatebag.rules.profile
+
+import java.net.URI
+import java.nio.charset.StandardCharsets
+
+import better.files.File
+import nl.knaw.dans.easy.validatebag._
+import nl.knaw.dans.easy.validatebag.rules.metadata.normalizeLicenseUri
+import nl.knaw.dans.lib.error._
+import org.apache.commons.configuration.PropertiesConfiguration
+
+class NumberedRulesSpec extends TestSupportFixture with CanConnectFixture {
+
```

Comment:
The checks that this test does are OK, but there are several aspects of the test that I am not happy with:

* You are creating the complete application object, although you are never running it in the test, so this is unnecessary. 
* You are using the CanConnectFixture, but when in the test do you need to connect to the internet?
* You are using the "debug-config" files. Those are intended for manual tests, though. Using them in a unit tests limits the freedom of someone who want to change the debug-config files a result of manual testing. Use the `src/main/assembly/dist/cfg` files instead of create a specific copy for the test, if you need those files in your test at all! (In this case you do not.)

A better solution would be to refactor the application so that the `allRules` map is external to the App-object and gets passed in. Then you can check the allRules map without having to create an App-object at all, and no need to bother yourself with the configuration.

Reply:
Done. I create now the rules in the unit test by creating xml-validators with null schema.